ast/ast.go

```diff
@@ -64,6 +64,10 @@ type ExprNode interface {
 	SetValue(val interface{})
```

Comment:
When Eval() buildin function, it is hard avoid to use interface right now, it will be removed in futher commit.